content/blog/2018/istio-authorization/index.md

```diff
@@ -0,0 +1,210 @@
+or “who can access method `getBook` in my `bookstore` service”. It is not designed to control access to application-specific
+resource instances, like access to “storage bucket X” or access to “3rd book on 2nd shelf”. Today this kind of application
+specific access control logic needs to be handled by the application itself.
+
+### Role Based Access Control + Conditions
+
+Istio authorization is Role-Based Access Control (RBAC) system. Compared to Attribute-Based Access Control (ABAC),
+RBAC has the following advantages:
+
+* **Roles allow grouping of attributes.** Roles are groups of permissions, which specifies the actions you are allowed
+to perform on a system. Users are grouped based on the roles within an organization. You can define the roles and reuse
+them for different cases.
+
+* **It is easier to understand and reason about who has access.** The RBAC concepts maps naturally to the business concepts.
+For example, a DB admin may have all access to DB backend services, while a web client may only be able to view the
+frontend service.
+
+* **It reduces unintentional errors.** RBAC policies make the originally complicated security changes easy. You won't have
+duplicate configurations in multiple places and later forget to update some of them when you need to make changes.
+
+On the other hand, Istio authorization is not a traditional RBAC system. It also allows users to define **conditions** using
+[combination of Istio attributes](/docs/reference/config/authorization/constraints-and-properties/). This gives Istio
+authorization plenty of flexibility to express complex access control policies. In fact, **the “RBAC + conditions” model
+that Istio authorization adopts, has all the benefits an RBAC system has, and supports the level of flexibility that
+normally an ABAC system provides.** We will show some examples in the [Examples section](#examples).
+
+### High Performance
+
+Because of its simple semantics, Istio authorization is enforced on Envoy as a native authorization support. At runtime, the
+authorization decision is completely done locally inside an Envoy filter, without dependency to any external module.
+This allows Istio authorization to achieve high performance and availability.
+
+### Work With/Without Primary Identities
+
+Like any other RBAC system, Istio authorization is identity aware. In Istio authorization policy, there is a primary
+identity called `user`, which represents the principal of the client.
+
+In addition to the primary identity, you can also specify any conditions that define the identities. For example,
+you can specify the client identity as “user Alice calling from Bookstore frontend service”, in which case,
+you have a combined identity of the calling service (`Bookstore frontend`) and the end user (`Alice`).
+
+However, strongly authenticated identity is not required for using Istio authorization. Istio authorization work with or
```

Comment:
Maybe add a sentence to make it clear that it's strongly recommended to use Authorization together with Authentication. (I see this is mentioned in later examples but putting it here is much more easier to be found)

Reply:
Added a sentence emphasizing that it is recommended to use authentication with authorization.